Formula/m/mingw-w64.rb

```diff
@@ -98,14 +105,16 @@ def install
         --with-sysroot=#{arch_dir}
         --prefix=#{arch_dir}
         --with-bugurl=#{tap.issues_url}
-        --enable-languages=c,c++,fortran
+        --enable-languages=c,c++,objc,obj-c++,fortran,lto
+        --enable-lto
```

Comment:
If lto is in the languages, I do not think it is necessary to pass `--enable-lto`, is it?

Reply:
According to the GCC docs:
- `--enable-language=lto` is implied if `--enable-lto` is set
- `--enable-lto` is set by default

So it seems to be redundant to specify either.